tests/ui.rs

```diff
@@ -98,8 +114,89 @@ fn miri_config(target: &str, path: &str, mode: Mode, with_dependencies: bool) ->
         ..Config::rustc(path)
     };
 
+    if let Mode::Pass { rustc: true } = mode {
+        config.comment_defaults.base().add_custom("run", Run {
+            exit_code: 0,
+            output_conflict_handling: Some(|path, actual, errors, config| {
+                let path = path.with_file_name(
+                    path.file_name().unwrap().to_str().unwrap().replace(".run.", "."),
```

Comment:
Where does this `.run.` come from?

Reply:
This is so you get `foo.run.stderr` that does not conflict with `foo.stderr`, which is for stderr during compilation

Reply:
But this *removes* `.run.`

Is this to make sure that the output of the "run" is compared with the normal reference file for Miri?

Reply:
Yes